api/v1beta2/moduletemplate_types_test.go

```diff
@@ -11,101 +10,6 @@ import (
 	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
 )
 
-func Test_GetSemanticVersion(t *testing.T) {
```

Comment:
To Reviewer:

This GetSemanticVersion can be dropped since the `spec.version` validation is done on the CRD level, and we are not using annotation anymore.